backend/clubs/views.py

```diff
@@ -506,6 +517,37 @@ def get_serializer_class(self):
             return WritableClubFairSerializer
         return ClubFairSerializer
 
+    @action(detail=True, methods=["get"])
+    def get_events_user_info(self):
```

Comment:
This will result in the endpoint being called `/clubfairs/{clubid}/get_events_user_info/?format=json`, which is kind of verbose. I would name it something like `live`.